src/controllers/controllerengine.cpp

```diff
@@ -1440,6 +1440,23 @@ void ControllerEngine::softTakeover(QString group, QString name, bool set) {
 }
 
 /*  -------- ------------------------------------------------------
+     Purpose: Ignores the next value for the given ControlObject
+                This is used when an absolute physical control is changed to
```

Comment:
"This should be called after an absolute physical control is changed to operate on a different..."